test/edu-benefits/5490/config/previousBenefits.unit.spec.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/5490/config/form';
+
+describe('Edu 5490 benefitSelection -> previousBenefits', () => {
+  const { schema, uiSchema } = formConfig.chapters.benefitSelection.pages.previousBenefits;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          formData={{}}
+          schema={schema}
+          data={{}}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input');
+
+    expect(fields.length).to.equal(5);
+  });
+
+  it('should expand options conditionally', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    // Note: The following checks for specific fields. Alternatively, we could
+    //  check for a certain number of fields instead.
```

Comment:
I didn't know if one way was preferable to another, so I checked for specific fields to be expanded rather than a certain number of fields shown, but I'm good with either way.